docs/workshop.md

```diff
@@ -1330,6 +1330,116 @@ As a side note, we really encourage you to take the time to dig in the toolbox o
 
 ---
 
+# Lab 5 : AAD + RBAC
+
+In this Lab we will focus on securing connections to Azure Cache for Redis by replacing secrets (e.g. connection strings) with [AAD-integration and RBAC][redis-aad-auth].
+
+![Using AAD to connect to Azure Cache for Redis](./assets/azure-ad-token.png)
+
+The integration of AAD with Azure Cache for Redis as follows:

```

Comment:
Here are the steps to integrate Azure Cache for Redis with a managed AAD Authentication and Autorization :